@xen-orchestra/lite/src/stories/web-core/tree/tree-item-label.story.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      iconProp().preset(faServer),
+      prop('route').required().preset('dashboard').widget(),
+      prop('active').bool().widget(),
+      prop('no-indent').bool().help('Remove indentation based on component depth in the tree').widget(),
```

Comment:
Since this option has no effect in this case, I propose to remove the widget.

```suggestion
      prop('no-indent').bool().help('Remove indentation based on component depth in the tree'),
```